dotcom-rendering/src/lib/formatCount.ts

```diff
@@ -4,10 +4,10 @@ export const formatCount = (
 	if (count === undefined) return { short: '…', long: '…' };
 	if (count === 0) return { short: '0', long: '0' };
 
-	const countAsInteger = parseInt(count.toFixed(0), 10);
+	const countAsInteger = Math.floor(count);
 	const displayCountLong = Intl.NumberFormat('en-GB').format(countAsInteger);
 	const displayCountShort =
-		countAsInteger > 10000
+		countAsInteger > 10_000
```

Comment:
That’s a fair point, I guess the best approach would be to simply [enforce this with an ESLint rule](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/numeric-separators-style.md) and discuss there. I’ll remove it from this PR.

In terms of support, I’m **almost certain** that setting browsers targets means that it will be simplified in the bundled output.